code/HabitTracker/app/src/main/java/com/example/habittracker/activities/eventlist/CustomList.java

```diff
@@ -62,6 +69,8 @@ public CustomList(Context context, ArrayList<HabitEvent> events){
         super(context,0, events);
         this.events = events;
         this.context = context;
+        mStorage = FirebaseStorage.getInstance();
+        mDatabaseRef = FirebaseDatabase.getInstance().getReference("uploads");
```

Comment:
hey so. I noticed this "uploads" reference is not documented anywhere.
At least it's not in the schema. Maybe this is something that should be added but idk if this is more of a @Jie-F thing